proxy/round_tripper/proxy_round_tripper.go

```diff
@@ -255,6 +256,7 @@ func (rt *roundTripper) RoundTrip(originalRequest *http.Request) (*http.Response
 				slog.Float64("dns-lookup-time", trace.DnsTime()),
 				slog.Float64("dial-time", trace.DialTime()),
 				slog.Float64("tls-handshake-time", trace.TlsTime()),
+				slog.String("local-addr", trace.LocalAddr()),
```

Comment:
From reading your spec description on the other PR, it is not clear if the "extra_headers" property is only for access logs or not. 

❓ Based on my reading of it, I would've thought it would affect these logs too? Thoughts?

Reply:
Hm, in the past we didn't give much thought to the compatibility of non-access-logs, primarily because these are JSON and we expect people to parse it as such. Additional fields shouldn't cause any breakage in those cases. For example the timing info directly above this line is controlled by a feature flag in the access logs but unconditionally included in this log.

Reply:
> I could adjust the wording in the release spec to reflect that clearly.

Sounds good to me.